site/content/tech.md

```diff
@@ -0,0 +1,15 @@
+---
+title: 'tech'
+date: 2018-11-14T19:02:50-07:00
+draft: false
+---
+
+# [Victor Hugo](https://github.com/netlify-templates/victor-hugo)
+
+## A Hugo boilerplate for creating truly epic websites
+
+<img src="https://d33wubrfki0l68.cloudfront.net/30790d6888bd8af863fb2b5c33a7f337cdbda243/4e867/images/hugo-logo-wide.svg" style="width: 40%" />
+
+This is a boilerplate for using [Hugo](https://gohugo.io/) as a static site generator and [Webpack](https://webpack.js.org/) as your asset pipeline. Victor Hugo setup to use [PostCSS](http://postcss.org/) and [Babel](https://babeljs.io/) for CSS and JavaScript compiling/transpiling. This project is released under the [MIT license](LICENSE). Please make sure you understand its implications and guarantees.
+
+## Enjoy!! 😸
```

Comment:
This is cool, thanks for adding it!

Reply:
No problem. I wanted to make sure content was getting handled correctly so I added a few pages - figured this might be worth keeping.